plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/hms/TrinoHiveCatalog.java

```diff
@@ -375,6 +377,12 @@ public Table loadTable(ConnectorSession session, SchemaTableName schemaTableName
         return getIcebergTableWithMetadata(this, tableOperationsProvider, session, schemaTableName, metadata);
     }
 
+    @Override
+    public Map<SchemaTableName, List<ColumnMetadata>> tryGetColumnMetadata(ConnectorSession session, List<SchemaTableName> tables)
```

Comment:
Provide the default implementation in `TrinoCatalog`

Reply:
That's not how we maintain this interface.